SayTheirNames/Source/View/CallToAction/CallToActionCell.swift

```diff
@@ -33,7 +33,7 @@ final class CallToActionCell: UICollectionViewCell {
         $0.translatesAutoresizingMaskIntoConstraints = false
         $0.contentMode = .scaleAspectFill
         $0.clipsToBounds = true
-        
+        $0.isHidden = true
```

Comment:
nice work 👍  any chance we can put this behind a FeatureFlag? There's already a precedent used in other places in the code base.

Reply:
Sure I'll add the feature flag now

Reply:
Would something like this be fine `$0.isHidden = !FeatureFlags.callToActionCellImageShown`

Reply:
Done